lib/money_column/active_record_type.rb

```diff
@@ -0,0 +1,6 @@
+class MoneyColumn::ActiveRecordType < ActiveRecord::Type::Decimal
+  def serialize(money)
+    return nil unless money
+    money.to_d
```

Comment:
`super(money.to_d)` so we hook into whatever AR is providing for us (now or in the future)?